Proxy/src/main/kotlin/net/horizonsend/ion/proxy/managers/ReminderManager.kt

```diff
@@ -0,0 +1,61 @@
+package net.horizonsend.ion.proxy.managers
+
+import java.util.concurrent.TimeUnit
+import net.horizonsend.ion.common.database.collections.PlayerData
+import net.horizonsend.ion.proxy.IonProxy.Companion.Ion
+import net.md_5.bungee.api.ChatColor
+import net.md_5.bungee.api.chat.ClickEvent
+import net.md_5.bungee.api.chat.ClickEvent.Action.RUN_COMMAND
+import net.md_5.bungee.api.chat.HoverEvent.Action.SHOW_TEXT
+import net.md_5.bungee.api.chat.ComponentBuilder
+import net.md_5.bungee.api.chat.HoverEvent
+import net.md_5.bungee.api.chat.hover.content.Text
+
+object ReminderManager {
```

Comment:
im not sure i like this, but I have no better solution, so itll do for now

Reply:
It does its job